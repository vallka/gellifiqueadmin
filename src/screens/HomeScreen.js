import React from 'react';
import {
  Image,
  Platform,
  ScrollView,
  StyleSheet,
  Text,
  TouchableOpacity,
  View,
  Button, 
  ActivityIndicator
} from 'react-native';

import { MyButton, MyBackground } from '../components/MyCompo';

export default class HomeScreen extends React.Component {
  static navigationOptions = {
    header: null,
  };

  render() {
    return (
      <View style={styles.container}>
        <MyBackground />
        <View
          style={{
            flex: 1,
            backgroundColor: 'transparent',
            justifyContent: 'center',
          }}
        >
        <MyButton text='Orders' onPress={()=>this.props.navigation.navigate('Orders')}/>
        <MyButton text='Orders 2' onPress={()=>this.props.navigation.navigate('Orders')}/>
        <MyButton text='Orders 3' onPress={()=>this.props.navigation.navigate('Orders')}/>

        </View>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#00f',
  },
});

