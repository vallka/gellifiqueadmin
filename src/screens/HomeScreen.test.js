import React from 'react';
import {shallow} from 'enzyme';
import HomeScreen from './HomeScreen';


describe('HomeScreen Rendering', () => {
    it('should match to snapshot', () => {
        const component = shallow(<HomeScreen />)
        expect(component).toMatchSnapshot()
    });
});

