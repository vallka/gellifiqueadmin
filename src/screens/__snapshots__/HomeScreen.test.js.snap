// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeScreen Rendering should match to snapshot 1`] = `
<Component
  style={
    Object {
      "backgroundColor": "#00f",
      "flex": 1,
    }
  }
>
  <MyBackground />
  <Component
    style={
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <MyButton
      onPress={[Function]}
      text="Orders"
    />
    <MyButton
      onPress={[Function]}
      text="Orders"
    />
    <MyButton
      onPress={[Function]}
      text="Orders"
    />
  </Component>
</Component>
`;
